### ISSUE TITLE
Maintenance/Updates

### ISSUE THREAD

**Issue by jph** [sowderca/connectwise-ruby-sdk#10]
Hello. I'm wondering if this gem is being actively maintained? ConnectWise have recently published some large-scale changes to their API and it would be a shame to see a Gem like this not keep up with their advances.

I am most definitely interested in contributing, as I've actively used this gem for a while now, and I've started noticing that with newer versions of the API, some data is missing from the existing data modeling in this Gem.

Please let me know if I can be of any help. I would send pull req's directly however I'm hesitant to do this as it looks like Swagger-codegen is being used - so if anybody were to contribute, it would be much easier to edit the Swagger API defs than to actually write code :stuck_out_tongue: 

Thank you.

**Comment by jph** [comment]
I'd be happy to help out where I can. I'm using ConnectWise daily at the moment, so keeping this Gem updated is important to me.

I attempted to compile the newer Swagger specs from ConnectWise but ran into the problem that you appear to have solved, combining the spec files into one before generating the library.

Happy to do anything I can to assist.

**Comment by sowderca** [comment]
@jph I've added the script I've been using for fetching the API specs. The specs I used to download for ConnectWise manage could be located `https://marketplace.connectwise.com/docs/openapi/json/{API}.txt`, however those appear to be out of date. They have re-done their API docs which use a guid stored in redoc's state to generate the url for the spec so its a bit tougher to automate. I'm currently using the specs contained in the installation. `https://{SiteUrl}{ConnectWiseManageAPIVersion}/services/ApiTools/ApiDocumentation/json/{spec}.json`

I've added you to the repo in case you have any ideas on how to automate this a bit more or use the manual process.
